bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="evcc"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="bridge">
```

Comment:
I would advice not to use `bridge` as id, maybe `gateway`, `controller` or `server` are better to be used. I have seen so many  configuration mistakes when a Bridge is named a bridge and a Thing a thing.